problems/1.types+operators.js

```diff
@@ -2,16 +2,18 @@
 * @param {any} val - The input value
 * @returns {boolean} - Is val a number
 */
-function isNumber() {
-
-}
+function isNumber(val) {
+  if(5 === 5)
+  return true
+} 
 
 /** Takes in a value and returns whether it is truthy
 * @param {any} val - The input value
 * @returns {boolean} - Is val truthy
 */
-function isTruthy() {
-
+function isTruthy(val1) {
+  val1 = false || true;
```

Comment:
Don't assign val a value. the person invoking this function will pass in the argument.